src/router/routes/errorHandling.ts

```diff
@@ -22,10 +23,22 @@ export const applicationErrorMapper = (
         next(new createError.BadRequest(`Failed to parse request body as JSON resource. Error was: ${err.message}`));
         return;
     }
+    if (err instanceof TooManyConcurrentExportRequestsError) {
+        next(
+            new createError.TooManyRequests(
+                'There is currently too many concurrent export requests. Please try again later',
+            ),
+        );
+        return;
+    }
     next(err);
 };
 
 export const httpErrorHandler = (err: any, req: express.Request, res: express.Response, next: express.NextFunction) => {
+    if (err instanceof createError.TooManyRequests) {
+        const RETRY_AGAIN_IN_SECONDS = 15 * 60;
```

Comment:
Is 15 mins the suggested or how did we come up with that? Add a minor comment saying 15mins. 
```suggestion
        const RETRY_AGAIN_IN_SECONDS = 15 * 60; // 15 minutes
```

Reply:
Updated with the comment. I got 15 minutes from reading forums online where people did ETL jobs with Glue. It usually took them under 10 minutes, so I thought 15 minutes would be a safe number for the retry.